test/e2e/app-dir/metadata/metadata.test.ts

```diff
@@ -491,13 +492,25 @@ createNextDescribe(
             'https://example.com/opengraph/static/opengraph-image.png?b76e8f0282c93c8e',
         })
 
+        const ogImgUrl = new URL($('[property="og:image"]').attr('content'))
```

Comment:
the tests seem to pass for all, the image size info are preserved properly but the content is broken and this change fixed that. wonder we can have some other tests to identify the broken image?

Reply:
I may need to create an `IncrementalCache` Unit test instead of e2e.

Reply:
https://github.com/vercel/next.js/pull/48237/files#diff-82703b603cb326438b3132cd325923a76de5585b55f2cb51352b4bcc6912a839

Replaced by unit test.